tests/test_mlkit.py

```diff
@@ -185,34 +265,84 @@
 invalid_string_or_none_args = [0, -1, 4.2, 0x10, False, list(), dict()]
 
 
-def check_error(err, err_type, msg=None):
+# For validation type errors
+def check_error(excinfo, err_type, msg=None):
+    err = excinfo.value
     assert isinstance(err, err_type)
     if msg:
         assert str(err) == msg
 
 
-def check_firebase_error(err, code, status, msg):
+# For errors that are returned in an operation
+def check_operation_error(excinfo, code, msg):
+    err = excinfo.value
+    assert isinstance(err, exceptions.FirebaseError)
+    assert err.code == code
+    assert str(err) == msg
+
+
+# For rpc errors
+def check_firebase_error(excinfo, code, status, msg):
+    err = excinfo.value
     assert isinstance(err, exceptions.FirebaseError)
     assert err.code == code
     assert err.http_response is not None
     assert err.http_response.status_code == status
     assert str(err) == msg
 
 
-def instrument_mlkit_service(app=None, status=200, payload=None):
+def instrument_mlkit_service(status=200, payload=None, operations=False, app=None):
```

Comment:
You can modify this helper to handle the multiple requests too.

```
if isinstance(status, list):
   adapter = MockMultiRequestAdapter(...)
else:
   adapter = MockAdapter()
```

Reply:
done